### ISSUE TITLE
\Cassandra\Timeuuid::toDateTime() does not work

### ISSUE THREAD

**Issue by evll** [nano-interactive/cassandra-php-driver#63]
Calling this method  results in 
`php: symbol lookup error: /usr/local/lib/php/extensions/no-debug-non-zts-20220829/cassandra.so: undefined symbol: _Z20php_date_get_date_cev`

Code to reproduce:

```php
$uuidCs = new \Cassandra\Timeuuid(time());
$uuidCs->toDateTime();
```

**Comment by evll** [comment]
It's the same for `\Cassandra\Timestamp` as well

**Comment by CodeLieutenant** [comment]
Ill investigate today

**Comment by CodeLieutenant** [comment]
@evll, just one thing, is this 1.3.8 tag or 1.3.x Branch?

**Comment by evll** [comment]
I was using this version from master: `90cdfa6ed39265c09e6a03eca30805cc5d1e09c7`

**Comment by CodeLieutenant** [comment]
Dont use master branch

**Comment by evll** [comment]
Sorry, I meant this was the default 1.3.x branch

**Comment by CodeLieutenant** [comment]
Ill fix it today, till then you can use 1.3.8 tag, which works fine

**Comment by CodeLieutenant** [comment]
Can you provide you PHP version?

**Comment by CodeLieutenant** [comment]
@evll I made a fix, if you can please try it.

**Comment by evll** [comment]
I tried building with [this latest commit](https://github.com/he4rt/scylladb-php-driver/commit/0d329ebdd2d7425bed961448e6471bf0dcf34022), but I still get the same error.

**Comment by CodeLieutenant** [comment]
> I tried building with [this latest commit](https://github.com/he4rt/scylladb-php-driver/commit/0d329ebdd2d7425bed961448e6471bf0dcf34022), but I still get the same error.

Try to build the pull request, its #66

**Comment by evll** [comment]
Yes, it works correctly now, thanks!

**Comment by CodeLieutenant** [comment]
I'll have to do some check on that branch, but it will be today merged in 1.3.x branch, same for the other issue you opened.

Thank you very much for issues, and please if you find more, open them as soon as you can
